db/fixtures/miq_product_features.yml

```diff
@@ -6213,6 +6213,26 @@
       :feature_type: view
       :identifier: firmware_show
 
+# Guest Devices
```

Comment:
@blomquisg if these are a new menu item in the navigation, we normally do add them into product_features so a role can be customized to show or not show them.  If they are similar to the `Firmwares` item above, then I'd say yes.  If not, then I'm not sure as I am not familiar with this addition.